app/lib/three_scale/collection_representer.rb

```diff
@@ -18,4 +18,13 @@ def to_xml(options = {})
       xml_collection.new(self).to_xml(options.merge(root: representation_wrap))
     end
   end
+
+  module JSONCollection
+    extend ActiveSupport::Concern
+    include Representable::JSON::Collection
+
+    def to_json(options = {})
+      ThreeScale::Api::Collection.new(self).to_json(options)
```

Comment:
Why not extend `ThreeScale::Api::Collection` instead? Wouldn't that be the same?
And then, `ThreeScale::Api::Collection` would extend `extend ActiveSupport::Concern`.

Reply:
why extending an entire module to use just one method? for that imo it is better just to delegate that specific method.
extending is like inheritance.

Reply:
btw there is the extra complication that 'collection' and 'represented' do not behave the same well because for representers we sometimes use modules and sometimes classes and it is pretty different and this has to cover all cases